test/e2e/data/infrastructure-aws/cluster-template.yaml

```diff
@@ -0,0 +1,328 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: ${CLUSTER_NAME}
+  labels:
+    ccm: external
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+      - 10.1.0.0/16
+    services:
+      cidrBlocks:
+      - 10.152.183.0/24
+  controlPlaneRef:
+    apiVersion: controlplane.cluster.x-k8s.io/v1beta2
+    kind: CK8sControlPlane
+    name: ${CLUSTER_NAME}-control-plane
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta2
+    kind: AWSCluster
+    name: ${CLUSTER_NAME}
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta2
+kind: AWSCluster
+metadata:
+  name: ${CLUSTER_NAME}
+spec:
+  region: ${AWS_REGION}
+  sshKeyName: ${AWS_SSH_KEY_NAME}
+  bastion:
+    enabled: ${AWS_CREATE_BASTION}
+  controlPlaneLoadBalancer:
+    healthCheckProtocol: TCP
+  network:
+    vpc:
+      id: vpc-09dc405df98c78806
```

Comment:
does this mean we always have a fixed VPC deployed and assume that it exists?

Reply:
I needed this to try to set a security group that allows all incoming traffic. Cannot create a security group without specifying a VPC, so I created one.

Once we find the root cause of the networking issue we can remove this.